dex/testing/zec/harness.sh

```diff
@@ -0,0 +1,232 @@
+# Load prepared wallet if the files exist.
+################################################################################
+
+mkdir -p "${ALPHA_DIR}"
+mkdir -p "${BETA_DIR}"
+
+# mkdir -p ${ALPHA_DIR}/regtest
+# cp ${SOURCE_DIR}/alpha_wallet.dat ${ALPHA_DIR}/regtest/wallet.dat
+# mkdir -p ${BETA_DIR}/regtest
+# cp ${SOURCE_DIR}/beta_wallet.dat ${BETA_DIR}/regtest/wallet.dat
+
+cd ${NODES_ROOT} && tmux new-session -d -s $SESSION $SHELL
+
+################################################################################
+# Write config files.
+################################################################################
+
+# These config files aren't actually used here, but can be used by other
+# programs. I would use them here, but bitcoind seems to have some issues
+# reading from the file when using regtest.
+
+cat > "${ALPHA_DIR}/zcash.conf" <<EOF
+rpcuser=user
+rpcpassword=pass
+txindex=1
+port=${ALPHA_LISTEN_PORT}
+regtest=1
+rpcport=${ALPHA_RPC_PORT}
+exportdir=${SOURCE_DIR}
+# Activate all the things.
+nuparams=5ba81b19:1
+nuparams=76b809bb:1
+nuparams=2bb40e60:1
+nuparams=f5b9230b:1
+nuparams=e9ff75a6:2
+EOF
+
+cat > "${BETA_DIR}/zcash.conf" <<EOF
+rpcuser=user
+rpcpassword=pass
+txindex=1
+regtest=1
+rpcport=${BETA_RPC_PORT}
+exportdir=${SOURCE_DIR}
+nuparams=5ba81b19:1
+nuparams=76b809bb:1
+nuparams=2bb40e60:1
+nuparams=f5b9230b:1
+nuparams=e9ff75a6:2
+EOF
+
+################################################################################
+# Start the alpha node.
+################################################################################
+
+tmux rename-window -t $SESSION:0 'alpha'
+tmux send-keys -t $SESSION:0 "set +o history" C-m
+tmux send-keys -t $SESSION:0 "cd ${ALPHA_DIR}" C-m
+echo "Starting simnet alpha node"
+tmux send-keys -t $SESSION:0 "${DAEMON} -rpcuser=user -rpcpassword=pass \
+  -rpcport=${ALPHA_RPC_PORT} -datadir=${ALPHA_DIR} \
+  -debug=rpc -debug=net -debug=mempool -debug=walletdb -debug=addrman -debug=mempoolrej \
+  -whitelist=127.0.0.0/8 -whitelist=::1 \
+  -txindex=1 -regtest=1 -port=${ALPHA_LISTEN_PORT} -fallbackfee=0.00001 \
+  -printtoconsole; tmux wait-for -S alpha${SYMBOL}" C-m
+sleep 3
+
+################################################################################
+# Setup the beta node.
+################################################################################
+
+tmux new-window -t $SESSION:1 -n 'beta' $SHELL
+tmux send-keys -t $SESSION:1 "set +o history" C-m
+tmux send-keys -t $SESSION:1 "cd ${BETA_DIR}" C-m
+
+echo "Starting simnet beta node"
+tmux send-keys -t $SESSION:1 "${DAEMON} -rpcuser=user -rpcpassword=pass \
+  -rpcport=${BETA_RPC_PORT} -datadir=${BETA_DIR} -txindex=1 -regtest=1 \
+  -debug=rpc -debug=net -debug=mempool -debug=walletdb -debug=addrman -debug=mempoolrej \
+  -whitelist=127.0.0.0/8 -whitelist=::1 \
+  -port=${BETA_LISTEN_PORT} -fallbackfee=0.00001 -printtoconsole; \
+  tmux wait-for -S beta${SYMBOL}" C-m
+sleep 3
```

Comment:
I had to increase this to `sleep 30` because the rpc calls below would fail because the client was still "warming up".